content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -116,10 +116,10 @@ Prisma supports the following referential actions:
 
 If you do not specify a referential action, Prisma uses the following defaults:
 
-| Clause     | Optional relations | Mandatory relations |
-| :--------- | :----------------- | :------------------ |
-| `onDelete` | `SetNull`          | `Restrict`          |
-| `onUpdate` | `Cascade`          | `Cascade`           |
+| Clause     | Optional relations | Mandatory relations | [Implicit Many-to-Many](#caveats) |
+| :--------- | :----------------- | :------------------ | :---------------------|
+| `onDelete` | `SetNull`          | `Restrict`          | `Cascade`             |
+| `onUpdate` | `Cascade`          | `Cascade`           | `Cascade`             |
```

Comment:
I am unsure: Is this addition helpful, considering that the table is introduced as "if you do not specify..." which one can not do for many-to-many?

Reply:
I also don't think this should be in the table in this way, as 'implicit many-to-many' is a different type of thing to 'optional' and 'mandatory'. Maybe a note linking to the other section?

```suggestion
| Clause     | Optional relations | Mandatory relations |
| :--------- | :----------------- | :------------------ | 
| `onDelete` | `SetNull`          | `Restrict`          |
| `onUpdate` | `Cascade`          | `Cascade`           |

Note that implicit many-to-many relations will always use `Cascade` for both `onUpdate` and `onDelete` referential actions. See [Caveats](#caveats) for more details.
```

Reply:
One problem, which I spent a lot of times to reflect in the suggested change below, is that the referential actions are _not_ on the m-n relation side itself - but invisible on the join table between them. Not sure if that is reflected in your suggestion @keerlu.

Reply:
Yes, I'm just linking to the section below for the details.